docker/run_tests_in_docker.sh

```diff
@@ -17,8 +17,8 @@ set -v -e -x
 PS4="+ (run_tests_in_docker.sh): "
 
 DC="$(which docker-compose)"
-APP_UID="10001"
-APP_GID="10001"
+SOCORRO_UID=${SOCORRO_UID:-"10001"}
+SOCORRO_GID=${SOCORRO_GID:-"10001"}
```

Comment:
This fixes a thing I broke when I redid how uid/gid works when building images a couple of weeks ago. Oops.

Now running `make testshell` and running pytest in it works--it was kicking up permission errors for me.